src/lib/multiaddres.ts

```diff
@@ -398,30 +398,35 @@ export async function updateRelaysJSON({
   noxes,
 }: UpdateRelaysJSONArgs) {
   if (
-    typeof fluenceConfig?.relaysPath !== "string" ||
+    typeof fluenceConfig?.relaysPath === "undefined" ||
     envConfig?.fluenceEnv === undefined
   ) {
     return;
   }
 
-  if (isAbsolute(fluenceConfig.relaysPath)) {
-    commandObj.error(
-      `relaysPath must be relative to the root project directory. Found: ${
-        fluenceConfig.relaysPath
-      } at ${fluenceConfig.$getPath()}}`,
-    );
-  }
+  const relayPaths =
+    typeof fluenceConfig.relaysPath === "string"
+      ? [fluenceConfig.relaysPath]
+      : fluenceConfig.relaysPath;
 
   const relays = await resolveAddrsAndPeerIds({
     fluenceEnv: envConfig.fluenceEnv,
     maybeFluenceConfig: fluenceConfig,
     noxes,
   });
 
-  await writeFile(
-    join(resolve(projectRootDir, fluenceConfig.relaysPath), "relays.json"),
-    jsonStringify(relays),
-  );
+  for (const relayPath of relayPaths) {
+    if (isAbsolute(relayPath)) {
+      commandObj.error(
+        `relaysPath must be relative to the root project directory. Found: ${relayPath} at ${fluenceConfig.$getPath()}}`,
+      );
+    }
+
+    await writeFile(
+      join(resolve(projectRootDir, relayPath), "relays.json"),
+      jsonStringify(relays),
+    );
+  }
```

Comment:
A small improvement can be to use `splitErrorsAndResults` to find ALL absolute paths at once so you can show all of them at the same time to the user like this
`relayPath must contain only paths relative to the root project directory. Found: ${absolutePaths.join('\n')}`
and then do `await Promise.all(relativePaths.map((relativePath) => writeFile(...)))`

Reply:
Made the suggested improvement. Please check